sdk/identity/Azure.Identity/src/Credentials/ManagedIdentityCredentialOptions.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Identity
+{
+    /// <summary>
+    /// Options used to configure the <see cref="ManagedIdentityCredential"/>.
+    /// </summary>
+    public class ManagedIdentityCredentialOptions : TokenCredentialOptions
+    {
+        /// <summary>
+        /// Specifies the configuration for the managed identity.
+        /// </summary>
+        public ManagedIdentityId ManagedIdentityId { get; set; }
```

Comment:
Should this default to `ManagedIdentityId.SystemAssigned`? If not, should we make this required by making it a required ctor param?

Reply:
Good point - my intent was to make it explicit, so I'd lean towards the ctor.